ci/citest/acceptance-test/tests/crash_recovery_test.go

```diff
@@ -0,0 +1,43 @@
+// +build acceptance
+
+package tests
+
+import (
+	"strings"
+	"testing"
+	"time"
+)
+
+func TestCrashRecovery(t *testing.T) {
+	stdout, _ := RunCmdAndReportFail(t, "openvdc", "run", "centos/7/lxc", `{"interfaces":[{"type":"veth"}], "node_groups":["linuxbr"]}`)
+	instance_id := strings.TrimSpace(stdout.String())
+
+	WaitInstance(t, 5*time.Minute, instance_id, "RUNNING", []string{"QUEUED", "STARTING"})
+	RunSshWithTimeoutAndReportFail(t, executor_lxc_ip, "sudo lxc-info -n "+instance_id, 10, 5)
+
+	_, _ = RunCmdWithTimeoutAndReportFail(t, 10, 5, "openvdc", "stop", instance_id)
+	t.Log("Waiting for instance to become STOPPED...")
+	WaitInstance(t, 5*time.Minute, instance_id, "STOPPED", []string{"RUNNING", "STOPPING"})
```

Comment:
I don't think this accurately simulates a crash. This is stopping the instance gracefully and even waiting for OpenVDC to register it as stopped. It doesn't create the state a crash does where OpenVDC thinks an instance is running while it's really not.

A much more accurate simulation would be to not stop the instance at all but just kill the KVM process of the mesos-slave/executor node it's running on. Then boot up KVM again and recover from there. Why not do it that way?

Reply:
The idea was to test to see if sending a command such as "start" would actually boot the lxc container back up. If it did, then that means that the "connection" has  been re-established.
But yeah, I guess I could change it to look a bit more realistic.